client/src/components/Layout.tsx

```diff
@@ -42,6 +47,28 @@ function Layout(ownProps: Props) {
       <main className="container mx-auto dark:bg-primary-background grow">
         {!props.web3Error && props.web3Initialized && props.chainID && children}
         {props.web3Error && <p>{props.web3Error}</p>}
+        {props.networkSwitchRequired && (
+          <>
+            <p>
+              Wrong network, please connect to one of the following networks:
+            </p>
+            <ul>
+              {Object.entries(chains).map((chain) => (
+                <li>
+                  <Button
+                    className="bg-transparent mx-4 outline-none focus:outline-none"
+                    key={chain[0]}
```

Comment:
don't we need `key` on the `<li>` instead of having it on `<Button>`?

Reply:
Yep, you are correct!

Fixing